From 09d5520d910b63fba67bea1d8c71f5d426f345b7 Mon Sep 17 00:00:00 2001
From: "aperez@igalia.com"
 <aperez@igalia.com@268f45cc-cd09-0410-ab3c-d52691b4dbfc>
Date: Wed, 12 Jul 2017 18:42:29 +0000
Subject: [PATCH] [WTF] Failure to build when the compiler specifically targets
 ARMv8-A / defines __ARM_ARCH_8A__
 https://bugs.webkit.org/show_bug.cgi?id=174425

Reviewed by Michael Catanzaro.

* wtf/Platform.h: Also check for __ARCH_ARM_8A__ to detect ARMv8.

Signed-off-by: Adrian Perez de Castro <aperez@igalia.com>

git-svn-id: http://svn.webkit.org/repository/webkit/trunk@219415 268f45cc-cd09-0410-ab3c-d52691b4dbfc
---
 Source/WTF/wtf/Platform.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Source/WTF/wtf/Platform.h b/Source/WTF/wtf/Platform.h
index 44d929c333c..7dccb44fcbf 100644
--- a/Source/WTF/wtf/Platform.h
+++ b/Source/WTF/wtf/Platform.h
@@ -238,7 +238,8 @@
     || defined(__ARM_ARCH_7S__)
 #define WTF_ARM_ARCH_VERSION 7
 
-#elif defined(__ARM_ARCH_8__)
+#elif defined(__ARM_ARCH_8__) \
+    || defined(__ARM_ARCH_8A__)
 #define WTF_ARM_ARCH_VERSION 8
 
 /* MSVC sets _M_ARM */
-- 
2.13.3

